activities/codebase-stewardship/supporting-governance/governance-template.md

```diff
@@ -9,89 +9,89 @@ The text below can be used as a template for creating a governance file in a rep
+* resolving issues in development or conflicts between contributors
+* managing and planning releases
+* controlling rights to [codebase] assets such as source repositories
 
-*
-
-The technical steering team has the joint responsibility to:
-
-* Provide technical direction for the codebase
-* Maintain a technical roadmap, an architecture and coding principles
-* Resolve issues in development or conflicts between contributors
-* Managing and planning releases
-* Controlling rights to [codebase] assets such as source repositories
+Ideally, no one company or organization will employ a simple majority of the technical steering team.
 
-The technical steering team meets regularly. Their agenda includes review of the technical roadmap and issues that are at an impasse. The intention of the agenda is not to review or approve all patches. This is mainly being done through the process described in CONTRIBUTING.md [add link].
+Any active member of the community can request to become a technical steering team member by asking the technical steering team. Members are appointed by the current technical steering team members.
 
-If more than one technical steering team member disagrees with a decision of the technical steering team they may appeal to the product steering team who will make the ultimate ruling.
+### Working process
 
-Ideally, no one company or organization will employ a simple majority of the technical steering team.
+The technical steering team meets regularly [explain frequency]. Their agenda includes review of the technical roadmap and issues that are at an impasse. The intention of the agenda is not to review or approve all patches. This is mainly done through the process described in CONTRIBUTING.md [add link].
 
 ## Product steering team
 
-Responsibilities of the product steering team:
+The product steering team is responsible for:
 
-* Maintaining the mission, vision, values, and scope of the project
-* Collecting planned features and presenting them in a unified view
-* Refining the governance as needed
-* Making codebase level decisions
-* Resolving escalated project decisions when the sub-team responsible is blocked
-* Managing the [codebase] brand
-* Licensing and intellectual property changes
-* Controlling access to [codebase] assets such as hosting and project calendars
+* maintaining the mission, vision, values, and scope of the project
+* collecting planned features and presenting them in a unified view
+* refining the governance as needed
+* making codebase level decisions
+* resolving escalated project decisions when the sub-team responsible is blocked
+* managing the [codebase] brand
+* licensing and intellectual property changes
+* controlling access to [codebase] assets such as hosting and project calendars
 
-Members are appointed by the current product steering team members.
+Ideally, no one company or organization will employ a simple majority of the product steering team.
```

Comment:
We explicitly removed this from my draft. Do you remember why @bvhme ?

Reply:
If only we would have used Git commit messages :P

I think it was because it added extra content that can but might not be relevant but didn't have that much use.